translations/es-ES/events/dalinar-travels-to-azimir.md

```diff
@@ -1,5 +1,5 @@
 # Dalinar viaja a Azimir
-Dalinar viajó a Azimiar para convencer a los azish de que se unieran a la coalición.
+Dalinar viajó a Azimiar para convencer a los azishianos de que se unieran a la coalición.
```

Comment:
Changed this as per description in `/translations/es-ES/locations/azir.md`
```
Tanto el nativo como el idioma de Azir reciben el nombre de azishiano.
```
I assume the demonym would be consistent with the adjective.